server/app.js

```diff
@@ -417,9 +419,10 @@ const templateFn = require('./../client/template.js');
 const renderPage = async (req, res)=>{
 	// Create configuration object
 	const configuration = {
-		local       : isLocalEnvironment,
-		publicUrl   : config.get('publicUrl') ?? '',
-		environment : nodeEnv
+		local            : isLocalEnvironment,
+		publicUrl        : config.get('publicUrl') ?? '',
+		environment      : nodeEnv,
+		codeMirrorThemes : codeMirrorThemes

```

Comment:
Can we try to consistently name these related variables some variation of "editorThemes" instead of "codeMirrorThemes", "themes", etc.